xbmc/addons/include/xbmc_pvr_types.h

```diff
@@ -244,6 +244,7 @@ extern "C" {
   {
     char         strGroupName[PVR_ADDON_NAME_STRING_LENGTH]; /*!< @brief (required) name of this channel group */
     bool         bIsRadio;                                   /*!< @brief (required) true if this is a radio channel group, false otherwise. */
+    int          iPosition;                                  /*!< @brief (optional) order position of the group */
```

Comment:
I think we best use an `unsigned int` instead here, and default to 0 when the add-on doesn't provide this feature (which is what they should be doing with `memset()` before assigning anyway)

then we can use some other sorting inside Kodi if it's set to 0, e.g. sorting by name